lib/modules/manager/maven/extract.spec.ts

```diff
@@ -173,5 +173,13 @@ describe('modules/manager/maven/extract', () => {
         'https://repo.adobe.com/v4/nexus/content/groups/public',
       ]);
     });
+
+    it('extract registries from a settings file that uses updated schema', () => {
+      const settingsUpatedContent = Fixtures.get(`settings.updated.schema.xml`);
```

Comment:
```suggestion
      const settingsUpatedContent = codeBlock`
<settings xmlns="http://maven.apache.org/SETTINGS/1.2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.2.0 http://maven.apache.org/xsd/settings-1.2.0.xsd">
  <mirrors>
        <mirror>
            <id>Test-Internal-repository</id>
            <name>Proxy Repository Manager</name>
            <url>https:/proxy-repo.com/artifactory/apache-maven</url>
            <mirrorOf>central</mirrorOf>
        </mirror>
    </mirrors>
  <profiles/>
  <activeProfiles/>
</settings>
`
```
This is what `@viceice` has been referring to.